iseq.c

```diff
@@ -835,13 +835,20 @@ iseq_setup_coverage(VALUE coverages, VALUE path, const rb_ast_body_t *ast, int l
     return Qnil;
 }
 
-rb_iseq_t *
-rb_iseq_new_top(const rb_ast_body_t *ast, VALUE name, VALUE path, VALUE realpath, const rb_iseq_t *parent)
+static inline void
+iseq_new_setup_coverage(VALUE path, const rb_ast_body_t *ast, int line_offset)
 {
     VALUE coverages = rb_get_coverages();
+
     if (RTEST(coverages)) {
         iseq_setup_coverage(coverages, path, ast, 0);
```

Comment:
@ioquatix Maybe you meant `line_offset` instead of `0`? The parameter is not used. (I would be happy if you'll give me time to review it :-)

Reply:
Thanks, yeah it looks like a mistake. Sorry about not giving enough time for a review, I appreciate your feedback.

Reply:
https://github.com/ruby/ruby/pull/7177